versioned_docs/version-1.2-rc1(WIP)/layer2Account.md

```diff
@@ -0,0 +1,39 @@
+---
+id: layer2Account
+title: Layer 2 Account in Godwoken
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+Godwoken v1 introduced the registry and registry address, both of which can be seen as alias mechanisms for layer 2 accounts. 
+
+A layer 2 account can be referenced by ID or by script (a hash of a script usually) itself. However, in some environments, it is also required to access an account by alias. For example, there is no such thing as a Godwoken `ID` or `script` in the EVM environment, and so access to an account can only be made through an ETH address.
```

Comment:
A layer 2 account can be referenced by an ID or by a script (usually a hash of a script). However, in some environments, it is also required to access an account by an alias. For example, in the EVM environment, there is no such thing as a Godwoken `ID` or `script`. Therefore, access to an account can only be made through an ETH address.